### PR TITLE
allowed_cidr_blocks support

### PR DESCRIPTION
**Pull request by arischow**
## what
* Added a new input named "allowed_cidr_blocks" for setting a ingress security group rule to allow a specific CIDR block to access the resource.

## why
* More convenient than setting up a extra security group by using other TF modules.

## references
* [terraform-aws-rds](https://github.com/cloudposse/terraform-aws-rds/blob/0a51b169b10ff2d42e99bbf2d9fb3fcf05799b1e/main.tf#L156-L165)

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -118,3 +118,9 @@ variable "transition_to_ia" {
   description = "Indicates how long it takes to transition files to the IA storage class. Valid values: AFTER_7_DAYS, AFTER_14_DAYS, AFTER_30_DAYS, AFTER_60_DAYS and AFTER_90_DAYS"
   default     = ""
 }
+
+variable "allowed_cidr_blocks" {
+  type        = list(string)
+  default     = []
+  description = "The whitelisted CIDRs which to allow `ingress` traffic to the DB instance"
```
Comment (nitrocode) [inline variables.tf:125]: One thing that the industry is trying to do is remove or replace terms like `whitelisted` and `blacklisted`. 

```suggestion
  description = "The CIDRs which to allow `ingress` traffic to the DB instance"
```

In this case, we can omit the word and maintain the same clarity.

**Comment by arischow2seedlink** [comment]
@maximmi If you could take 5 minutes to check this out

**Comment by arischow** [comment]
@maximmi It would be appreciated if you could take 5 minutes to check this out 😃

File: variables.tf [hunk 2]
```diff
@@ -118,3 +118,9 @@ variable "transition_to_ia" {
   description = "Indicates how long it takes to transition files to the IA storage class. Valid values: AFTER_7_DAYS, AFTER_14_DAYS, AFTER_30_DAYS, AFTER_60_DAYS and AFTER_90_DAYS"
   default     = ""
 }
+
+variable "allowed_cidr_blocks" {
+  type        = list(string)
+  default     = []
+  description = "The CIDRs which to allow `ingress` traffic to the DB instance"
```
Comment (maximmi) [inline variables.tf:125]: Please, fix the description to use EFS instead of DB instance. Looks like its just a copy/paste

**Comment by arischow** [comment]
@maximmi You caught me & fixed.

**Pull request closed without merging**